filament/src/driver/DataReshaper.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef TNT_FILAMENT_DRIVER_DATARESHAPER_H
+#define TNT_FILAMENT_DRIVER_DATARESHAPER_H
+
+#include <stddef.h>
+
+namespace filament {
```

Comment:
Should we nest this under the `driver` namespace?

Reply:
Well, the other shared classes in this folder don't seem to live in the driver namespace, but I could go either way.

Reply:
True- in that case it's probably fine as is.